apis/dns/v1alpha1/managed_zone_types.go

```diff
@@ -0,0 +1,129 @@
+/*
+Copyright 2022 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+)
+
+// ManagedZoneParameters define the desired state of a ManagedZone
+type ManagedZoneParameters struct {
```

Comment:
I saw that you did not implemented all possible fields in the ManagedZone and that's fine. I would just recommend you to add some comments where those fields should be created.

For example:

```golang
type ManagedZoneParameters struct {

// TODO(danielinclouds): PeeringConfig

...
```

That's just to people in future that are searching for the fields of the ManagedZone see which fields are missing implementation. Check [this example](https://github.com/crossplane-contrib/provider-gcp/blob/d24f532f4a4e794efa88325601770e144d8d8fa4/apis/container/v1beta2/cluster_types.go#L1165).

Reply:
fixed